src/main/webapp/product.html

```diff
@@ -90,7 +90,7 @@ <h2 class="businessName">
           Reviews for 
         </h2>
       </div>
-      <div class="review-form">
+      <div class="review-form" style="visibility: hidden;">
```

Comment:
I think you can add this in the else statement under not logged in right?

Reply:
Is there any benefit to doing that instead of just setting it as hidden by default in the html?

Reply:
I think the difference is that you are manipulating styling dynamically instead of html directly ba! Up to you then!